src/portfolios/portfolio/components/shared/PortfolioSummaryPageHead.vue

```diff
@@ -321,13 +322,13 @@ export default class PortfolioSummaryPageHead extends Vue {
   
   public async openSlideoutPanel(e: Event): Promise<void> {
     const currentSlideoutComponent = SlideoutPanel.slideoutPanelComponent;
-    if (e && e.currentTarget) {
+    if (e?.currentTarget) {
       e.preventDefault();
-      e.cancelBubble = true;
```

Comment:
cancelBubble is depreciated.  https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelBubble